modules/multi-architecture-import-imagestreams.adoc

```diff
@@ -0,0 +1,72 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="multi-architecture-import-imagestreams_{context}"]
+
+= Importing manifest lists in image streams on your multi-architecture cluster 
+
+On a {product-title} {product-version} multi-architecture cluster, the image streams in the cluster do not import manifest lists automatically. You must manually change the default `importMode` option to the `PreserveOriginal` option in order to import the manifest list.
+
+[NOTE]
+====
+For this procedure to run successfully, the `referencePolicy.type` field of your `ImageStream` object must be set to the `Source` type. 
+[source,yaml]
+----
+referencePolicy:
+    type: Source 
+----
+====
+
+.Prerequisites 
+
+* You installed the Openshift CLI (`oc`).
+
+.Procedure 
+
+* You can import the manifest list by updating the existing image stream tag. Use the following example cli artifacts to change the default `importMode` option and update the existing `ImageStream` file:
```

Comment:
> The following example shows how to patch the ImageStream cli-artifacts so that the ImageStreamTag cli-artifacts:latest  is imported as a ManifestList.

Reply:
Thank you Alessandro! What do you think of this wording?

Reply:
i don't think the wording is accurate Kelly. what Alessandro has is correct and reflects that when we patch the imagestream, we are patching a specific tag..so in that sense the `cli-artifacts:latest` tag is updated to import a manifestlist. the `latest` is not a type but an imagestream tag. an imagestream can have multiple tags. so , for example the user in this case could also add his/her own tag and set importMode to manifestlist in that tag and that specific tag would reflect a manifestlisted image whereas the other tags would be untouched

Reply:
Gotcha! The main thing I was trying to do was expand the `ImageStreamTag cli-artifacts:latest` statement becuase I now get that ImageStreamTag is describing the cli-artifact:latest but I think I got lost in which ones were objects and which one just needed to have a noun to describe it. 

I think I understand better now, thank you! Hmm, what about this: 

> The following example command shows how to patch the `ImageStream` cli artifacts so that the `cli-artifacts:latest` image stream tag is imported as a manifest list

Reply:
that sounds good!

Reply:
Just another dash between cli and artifacts: `how to patch the ImageStream cli-artifacts`.